syntax = "proto3";

message createDelete_request{
  string cacheName = 1;

}

message createDelete_response{
  string error = 1;
  string response = 2;

}

service cache{
  rpc createCache(createDelete_request) returns(createDelete_response){}

  rpc deleteCache(createDelete_request) returns(createDelete_response){}

}